lms/djangoapps/mobile_api/social_facebook/courses/tests.py

```diff
@@ -11,6 +14,7 @@
 from ..test_utils import SocialFacebookTestCase
 
 
+@patch.dict(settings.FEATURES, {"CUSTOM_COURSES_EDX": False})
```

Comment:
Once you remove the `unicode` conversion (see earlier comment), this should be addressed.  And then you should be able to not patch this setting.
Remember, this new code needs to work when CUSTOM_COURSES_EDX is enabled (which it will be on prod).